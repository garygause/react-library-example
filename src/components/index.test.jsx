import renderer from 'react-test-renderer';
import { describe, expect, it } from 'vitest';
import Hello from './index';

describe('Hello component', () => {
  it('Hello component renders correctly', () => {
    const component = renderer.create(<Hello />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(' prop working', () => {
    const component = renderer.create(<Hello message={'Friend'} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
