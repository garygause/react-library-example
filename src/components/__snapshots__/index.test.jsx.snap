// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Hello component >  prop working 1`] = `
<div>
  Hello, 
  Friend
  !
</div>
`;

exports[`Hello component > Hello component renders correctly 1`] = `
<div>
  Hello, 
  World
  !
</div>
`;
